Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs

```diff
@@ -42,28 +42,43 @@ static InputActionEditorWindow()
             InputActionAssetEditor.RegisterType<InputActionEditorWindow>();
         }
 
+        // Unity 6.4 changed signature of OpenAsset, and now it accepts entity id instead of instance id.
+        [OnOpenAsset]
+#if UNITY_6000_4_OR_NEWER
+        public static bool OpenAsset(EntityId entityId, int line)
+        {
+            if (!InputActionImporter.IsInputActionAssetPath(AssetDatabase.GetAssetPath(entityId)))
```

Comment:
Now that I read this code myself, I could have left IsInputActionAssetPath in OpenAsset, just the GetAssetPath thing is different. IDK, either way goes fine by my books

Reply:
Minor: But I would agree with you, that if it makes the "fork condition" slimmer it is a good idea.

Reply:
But its also fine as-is